setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import find_packages, setup
+
+with open('requirements.txt') as f:
+    requirements=f.read().splitlines()
+
+setup(
+    name="spog-ui-tests",
+    version="0.0.1",
+    author="RHTC QE team",
+    author_email="rhtc-qe@redhat.com",
+    description="UI Tests for Trustification SPoG",
+    url="https://github.com/trustification/spog-test.git",
+    pakages=find_packages(),
```

Comment:
@mrrajan how does the find_packages() function work? According to https://www.geeksforgeeks.org/what-is-setup-py-in-python/ the 'packages' parameter should include the package you need to install and the 'install_requires' parameter should only contain the package's dependencies.

Reply:
find_packages() simply retrieves all the functions from the root directory and it is useful for the package and distribution. I am not sure about the article though, may be they want to cherry pick the packages which are only to be installed on the client. 
install_requires is for more basic packages which needs to be installed. Here I point it to requirements file. We could change it in the future for specific dependencies in future.